util.py

```diff
@@ -87,6 +87,17 @@ def existingsnippet_monday(today):
     return end_monday.date()
 
 
+def _backfill_missing_snippets(user, all_snippets,
+                               current_monday, first_allowed_monday):
+    monday_ptr = current_monday - datetime.timedelta(7)
+    while monday_ptr - first_allowed_monday >= datetime.timedelta(0):
+        all_snippets.insert(0,
+            Snippet(email=user.email, week=monday_ptr,
```

Comment:
This indentation is weird -- I'd have expected your editor to line up `Snippets` with the paren.

I think the easiest solution is to just move the `0` to a line of its own, then it can line up with the existing `Snippet`.

Reply:
I ran `FormatLine` on this function. It does not exceed length limit now.